memory-bank/activeContext.md

```diff
@@ -0,0 +1,27 @@
+# Active Context: Pagination Implementation
```

Comment:
Please, put these files inside some subfolder (paginatedTable) or near paginated table itself. Currently it may be misinterpreted at least by the human - I expect docs in `/memoryBank/systemPatters` to be related to the whole project

Reply:
Not sure if we need this in our main